.gitignore

```diff
@@ -0,0 +1 @@
+config.py
```

Comment:
Add a newline at the end of the file. It is indicated by ![image](https://user-images.githubusercontent.com/30315706/101144681-b2c41800-363e-11eb-9282-87d2b610e654.png) in the PR info.

Reply:
Where do I need to write updater.idle() ? In the .gitignore file or  In bot.py